regression-tests.auth-py/test_LuaRecords.py

```diff
@@ -443,6 +444,19 @@ def testIfportupWithAllDownAndAllBackupSelector(self):
         self.assertRcodeEqual(res, dns.rcode.NOERROR)
         self.assertEqual(res.answer, expected)
 
+    def testIfportupWithAllDownAndEmptyBackupSelector(self):
+        """
+        Basic ifportup() test with all ports DOWN, fallback to 'empty' backup selector
+        """
+        name = 'empty.ifportup.example.org.'
```

Comment:
and of course the matching change
```suggestion
        name = 'empty.noneup.ifportup.example.org.'
```